_episodes_rmd/01-starting-with-data.Rmd

```diff
@@ -430,7 +430,7 @@ We'll learn why this is so in the next lesson.
 > > ## Solution
 > > ~~~
 > > whichPatients <- seq(2,40,2)
-> > whichDays <- c(1:5)
+> > whichDays <- 1:5
```

Comment:
Yes it is redundant, but keeping the `c()` serves as another visual cue to denote a vector.
My concern is that new learners may get confused when we simplify the code too much